airflow/bin/airflow

```diff
@@ -1,9 +1,30 @@
 #!/usr/bin/env python
 import logging
 import os
+import socket
+import requests
 from airflow import configuration
 from airflow.bin.cli import CLIFactory
 
+def get_private_ip(name=''):
+    r = requests.get("http://169.254.169.254/latest/meta-data/local-ipv4")
```

Comment:
Yeah, provisioning is aware of it, and unfortunately it seems that this is by design. Lyft uses the hostname to encode some information about the host that's used for monitoring purposes, see the previous discussions in these two Slack threads:

1. https://lyft.slack.com/archives/C3ASS377S/p1493405313083642
2. https://lyft.slack.com/archives/C2U75K0H5/p1484849237000002